src/sentry/integrations/vsts/integration.py

```diff
@@ -165,6 +168,15 @@ def get_organization_config(self):
             },
         ]
 
+        organization = Organization.objects.get(id=self.organization_id)
+        has_issue_sync = features.has('organizations:integration:issue_sync',
+                                      organization)
+        if not has_issue_sync:
+            for field in fields:
+                field['disabled'] = True
```

Comment:
We could also just completely turn off configuration for Jira by retuning an empty list of fields

Reply:
i kinda like letting them see what the features are